gradio/components.py

```diff
@@ -3637,7 +3637,6 @@ def update(
         label: str | None = None,
         show_label: bool | None = None,
         visible: bool | None = None,
-        interactive: bool | None = None,
```

Comment:
I guess this never worked so it's not a breaking change 😂 ?

Reply:
I suppose there is a *tiny* possibility someone wrote code with update(interactive=something) but I think that is outweighed by the benefit of having accurate type hints